src/schema/artwork/index.ts

```diff
@@ -692,7 +693,7 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
       },
       series: markdown(),
       show: {
-        type: PartnerShow.type,
+        type: Show.type,
```

Comment:
This is a breaking change that I suspect will break clients out in the wild that rely on it. We should really make this change for v2 (please file a ticket for that), but perhaps for v1 we should just leave this as-is?